kernel-modules/support-packages/04-create-support-packages.sh

```diff
@@ -60,6 +60,9 @@ for mod_ver_dir in "${MD_DIR}/module-versions"/*; do
     compress_files "$package_root" "${package_out_dir}/${filename}"
     generate_checksum "${package_out_dir}" "${filename}"
 
+    # Export the latest build filename
+    echo "${filename}" > "${package_out_dir}/latest"
```

Comment:
I think this might actually be a little more useful, since support packages are split by mod version and it would make it easier when creating the download URL
```suggestion
    echo "${mod_ver}/${filename}" > "${package_out_dir}/latest"
```

Reply:
I think it's not needed since `package_out_dir="${OUT_DIR}/${mod_ver}"` <- already contains the mod version. We need the mod version anyway to fetch the `latest` file. And it's easy to concatenate it to generate the full URL.